### DISCUSSION TITLE
FastAPI app version same as poetry pyproject version

### DISCUSSION THREAD

**Discussion by softwarebloat** [fastapi/fastapi#6393] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
no code
```

### Description

Hi all!
I just noticed that the FastAPI class has a `version` field with a default value set to `0.1.0`
how can i set it to match the pyproject version? is there a clean way of doing so?🤔
cheers

### Operating System

Linux

### FastAPI Version

0.75.2

### Python Version

3.9.0

**Comment by n8sty** [comment, marked as answer]
I'm assuming you mean the version value from your project's `pyproject.toml`. You can pass a version string to `FastAPI` like so:
```
from fastapi import FastAPI
FastAPI(version=your_package.version)
```
How you set `your_package.version` or `your_package.__version__` is up to you, but isn't a question for FastAPI.

**Comment by softwarebloat** [comment]
yeah i think you are right. it is not up to FastAPI
using it i was wondering how others do that without thinking is not a FastAPI related issue

Thanks anyway :beers:
